examples/debug/debug.sh

```diff
@@ -0,0 +1,145 @@
+#!/bin/bash
+
+# This script launches a job pod in a persistent state using 'sleep infinity', 
+# allowing us to utilize the Kubespray environment within this pod for debugging Ansible playbooks.
+# 
+# 1. apply kubean custom resources and create pod.
+# ./debug.sh
+#
+# 2. enter the kubespray container environment.
+# kubectl -n kubean-system exec -it kubean-debug-install-ops-job-xxxxx bash
+#
+# 3. get kubespray-related operation commands inside the container.
+# cat /bin/entrypoint.sh
+#
+# 4. delete custom resources.
+# kubectl delete cluster debug
+#
+
+DEBUG_NODE_ADDR="172.168.10.66"
+DEBUG_NODE_USER="root"
+DEBUG_NODE_PASS="1q2w3e4r"
+
+KUBEAN_VERSION="v0.13.4"
+
+CLUSTER_NAME="debug"
+ 
+cat << EOF | kubectl apply -f -
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: ${CLUSTER_NAME}-hosts-conf
+  namespace: kubean-system
+data:
+  hosts.yml: |
+    all:
+      hosts:
+        node1:
+          ip: ${DEBUG_NODE_ADDR}
+          access_ip: ${DEBUG_NODE_ADDR}
+          ansible_host: ${DEBUG_NODE_ADDR}
+          ansible_connection: ssh
+          ansible_user: ${DEBUG_NODE_USER}
+          ansible_password: ${DEBUG_NODE_PASS}
```

Comment:
Will `pubkey` login be supported?

Reply:
This script is intended to provide a simple debugging environment only in non-production environments. Using pubkey would involve operations such as key generation and adding secrets, which would complicate the script. It is not currently being considered unless there is a valid reason to justify the necessity.